intro-HPC/ch_MATLAB.tex

```diff
@@ -0,0 +1,80 @@
+\chapter{MATLAB}
+\label{ch:matlab}
+
+\section{Why is the MATLAB compiler required?}
+
+The main reason behind this alternative way of using MATLAB is licensing: only
+a limited number of MATLAB sessions can be active at the same time. However, once
+the MATLAB program is compiled using the MATLAB compiler, the resulting stand-alone
+executable can be run without needing to contact the license server.
```

Comment:
Include a reference to https://nl.mathworks.com/help/compiler/index.html, and mention that a license is required for the MATLAB Compiler (it's an additional package on top of MATLAB).